reference/sql/statements/split-region.md

```diff
@@ -173,6 +186,155 @@ region3  [("b", "2019-04-17 14:26:19") , ("c", "")                   )
 region4  [("c", "")                    , maxIndexValue               )
 ```
 
+### Split Regions for partitioned tables
+
+Splitting Regions for partitioned tables is the same as splitting Regions for normal tables. The only difference is that the split operation is performed for every partition.
+
++ The syntax of even split:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SPLIT [PARTITION] TABLE t [PARTITION] [(partition_name_list...)] [INDEX index_name] BETWEEN (lower_value) AND (upper_value) REGIONS region_num
+    ```
+
++ The syntax of uneven split:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    SPLIT [PARTITION] TABLE table_name [PARTITION (partition_name_list...)] [INDEX index_name] BY (value_list) [, (value_list)] ...
+    ```
+
+#### Examples
+
+Create a partitioned table `t`:
```

Comment:
More commits are made to https://github.com/pingcap/docs-cn/pull/2810. Please refer to the latest version and update this PR accordingly.